pkgs/by-name/ni/nixfmt-rfc-style/generated-package.nix

```diff
@@ -0,0 +1,31 @@
+# This file has been autogenerate with cabal2nix.
+# Update via ./update.sh
+{ mkDerivation, base, cmdargs, directory, fetchzip, filepath, lib
+, megaparsec, mtl, parser-combinators, safe-exceptions, scientific
+, text, transformers, unix
+}:
+mkDerivation {
+  pname = "nixfmt";
+  version = "0.5.0";
+  src = fetchzip {
+    url = "https://github.com/piegamesde/nixfmt/archive/0a8c246723bdd16207bb03681614892fa1d2f9b5.tar.gz";
+    sha256 = "029icwdl3gvmg377xyw8904yz0d05v0n2ksbf3qclgrq27v4a65x";
+  };
+  isLibrary = true;
+  isExecutable = true;
+  libraryHaskellDepends = [
+    base megaparsec mtl parser-combinators scientific text transformers
+  ];
+  executableHaskellDepends = [
+    base cmdargs directory filepath safe-exceptions text unix
+  ];
+  jailbreak = true;
+  homepage = "https://github.com/serokell/nixfmt";
+  description = "An opinionated formatter for Nix";
+  license = lib.licenses.mpl20;
+  mainProgram = "nixfmt";
+  maintainers = [
+    lib.maintainers._0x4A6F lib.maintainers.infinisil
```

Comment:
Maybe I'd be wise to add a team to teams-list.nix? That way it's both more visible who is in the team (and it can even be pinged as part of the release efforts :heart:) and if we get more packages, we can update the maintainer list with team members in one location

Reply:
Done, created the tentative `lib.teams.formatter` with all the members from the current RFC draft (though I can't add @Sereja313 since they're not a Nixpkgs maintainer), and assigned it as a maintainer of this package.